### PR TITLE
Wrote a common service for methods that will be used in both booking and cancellation; has 100% unit test coverage

### PR DESCRIPTION
**Pull request by justin-obrien-smoothstack**

### CODE REVIEW

File: src/main/java/com/ss/training/utopia/counter/service/UserService.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import com.ss.training.utopia.counter.entity.User;
+
+/**
+ * @author Justin O'Brien
+ */
+public class UserService {
+
+	@Autowired
+	UserDao userDao;
+
```
Comment (trevorhuis-smoothstack) [inline src/main/java/com/ss/training/utopia/counter/service/UserService.java]: great idea to be able to check for this.

**Review by trevorhuis-smoothstack — APPROVED** [review]

**Comment by justin-obrien-smoothstack** [comment]
Added another method and test.

**Comment by justin-obrien-smoothstack** [comment]
Removed DAO exception catching.

**Comment by justin-obrien-smoothstack** [comment]
Renamed from UserService to CommonService and added method to get all airports.

**Pull request merged**